CONTRIBUTING.md

```diff
@@ -1,5 +1,18 @@
 # Contributing guidelines
 
+## Contribution Checklist
+
+- [ ] Read the [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
Where does this link point to? We don't have a code of conduct file.

Reply:
on second thoughts I believe you are correct
I will make the necessary changes